maven-surefire-common/src/main/java/org/apache/maven/plugin/surefire/report/StatelessXmlReporter.java

```diff
@@ -387,7 +390,9 @@ private void startTestElement(XMLWriter ppw, WrappedReportEntry report) throws I
             ppw.addAttribute("classname", extraEscapeAttribute(className));
         }
 
-        ppw.addAttribute("time", report.elapsedTimeAsString());
```

Comment:
Can we remove method `elapsedTimeAsString` and used `org.apache.maven.plugin.surefire.report.ReporterUtils`?

Reply:
No, we can't. See here: https://github.com/apache/maven-surefire/pull/635/commits/9dab367d7e200022b4e88eb58cdfcf281824a124#diff-efbf1a613b0d3b854d7a732ab8f6221fa5ac34210449cdfb4eee6bb61a60c1ccR185
In fact, this subsequent commit makes `ReporterUtils` superfluous. My object was to clearly seperate serialization/deserialization of machine data (which are never locale-specific, but neutral), rendering happens in the view layer: https://github.com/apache/maven-surefire/pull/635/commits/9dab367d7e200022b4e88eb58cdfcf281824a124#diff-efbf1a613b0d3b854d7a732ab8f6221fa5ac34210449cdfb4eee6bb61a60c1ccR59-R61

Reply:
Yet another problem is that it was used only in one dir, not in both which is inconsistent and non-portable. I have basically applied what Ant does: https://github.com/apache/ant/blob/6f68fbdefad521741b37304cfdfc6d8646cf1839/src/main/org/apache/tools/ant/taskdefs/optional/junit/XMLJUnitResultFormatter.java#L189-L190

Reply:
ok